firmware/spade/README.md

```diff
@@ -103,7 +144,7 @@ export PICO_EXTRAS_PATH=~/raspberrypi/pico-extras
 
 ## Project Structure
 
-Spade uses CMake to build its binaries across platforms, although it's only tested to work on macOS.
+Spade uses CMake to build its binaries across platforms. It works best on MacOS and Linux, although Windows builds can work.
```

Comment:
I think it is best to not modify this line in this PR, as supported platforms for Spade is beyond the scope of this change.